_example/helper/field.go

```diff
@@ -165,16 +178,25 @@ func FieldToMap(model interface{}, fields map[string]interface{}) map[string]int
 
 		if contains(fields, jsonKey) {
 			v := fields[jsonKey]
+			var err error
```

Comment:
Thanks for your review! I fixed.

Since the mistake was found, it was fixed also.